.github/workflows/generate_workflows_lib/src/generate_workflows_lib/__init__.py

```diff
@@ -55,6 +55,14 @@ def get_test_job_datas(tox_envs: list, operating_systems: list) -> list:
         "py312": "3.12",
     }
 
+    # we enable windows testing only for packages with windows specific code paths
+    per_tox_env_os_enablement = {
+        "windows-latest": {
+            "py312-test-instrumentation-botocore",
+            "py312-test-instrumentation-system-metrics",
```

Comment:
Do we also need to care about running contrib tests in Windows for the core repo as well? For example, [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/.github/workflows/contrib_0.yml#L1673)

Reply:
I really hope changes in core that will affect -contrib are not platform dependenant :)